tools/travis/run_ppa_autopkgtests.sh

```diff
@@ -18,8 +18,15 @@
 
 set -ev
 
+lxc="/snap/bin/lxc"
+
 script_path="$(dirname "$0")"
-"$script_path/run_docker_container.sh" autopkgtest-requests
-docker exec -i autopkgtest-requests apt install -y python-launchpadlib
-docker exec -i autopkgtest-requests ./tools/run_ppa_autopkgtests.py
-docker rm -f autopkgtest-requests
+project_path="$(readlink -f "$script_path/../..")"
+
+"$script_path/setup_lxd.sh"
+"$script_path/run_lxd_container.sh" test-trigger
```

Comment:
FYI, when reviewing this I noticed that we're creating a privileged container because of LP: [#1709536](https://bugs.launchpad.net/snapd/+bug/1709536), which seems to have been fixed.